src/main/java/com/teamscale/buildbreaker/evaluation/Finding.java

```diff
@@ -0,0 +1,24 @@
+package com.teamscale.buildbreaker.evaluation;
+
+public class Finding {
```

Comment:
Newly written to have a better type for transferring data from the client to the evaluation classes. Before, we just passed the response string/JSON and parsing was done in the evaluators.